firestore-translate-text/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Version 0.1.7
+
+feature - added warsaw (europe-central2) location (#677)
```

Comment:
Same

Reply:
Realized we may not need a version update to this extension so reverting changes in this file.